src/resources/elements/typefield.js

```diff
@@ -92,18 +92,18 @@ export class Typefield extends Field {
   init(id = '', args = {}) {
     args = Object.assign({
       valueKey: '',
+      typeKey: 'x-oad-type',
       keyKey: '',
       keyPlaceholder: 'Object key...',
-      showType: true,
       copyValue: false,
       collapsed: false,
       types: { 'null': { 'type': 'text' } }
     }, args);
     this.types = args.types;
```

Comment:
Add line level comment describing why all of these values are mapped to `this`. Just a short comment clarifies why the code is doing what it is doing.